torch/testing/_internal/common_methods_invocations.py

```diff
@@ -7767,6 +7803,33 @@ def wrapper(x: np.ndarray, *args, **kwargs):
                # Topk is not raising a warning when the out is resized
                DecorateInfo(unittest.skip("Skipped!"), 'TestCommon', 'test_out'),
            )),
+    # Multiple variants for batch_norm to test with and without cuDNN disabled
+    # See https://github.com/pytorch/pytorch/pull/63218#discussion_r688549391 for more details
+    OpInfo('nn.functional.batch_norm',
+           aten_name='batch_norm',
+           dtypes=floating_types(),
+           dtypesIfCUDA=floating_types_and(torch.float16, torch.bfloat16),
+           supports_out=False,
+           skips=(
+               # RuntimeError: deepEquals(input.iValue, deepCopiedInput) INTERNAL ASSERT FAILED at
+               # "../torch/csrc/jit/passes/utils/check_alias_annotation.cpp":142, please report a bug to PyTorch
+               DecorateInfo(unittest.skip("Skipped!"), 'TestJit', 'test_variant_consistency_jit'),
+           ),
+           sample_inputs_func=sample_inputs_batch_norm),
+    # This variant tests batch_norm with cuDNN disabled on only CUDA devices
+    OpInfo('nn.functional.batch_norm',
```

Comment:
I'm not completely sure that OpInfo should be in the business of testing with cuDNN disabled -- this seems very backend specific. Feel free to leave this or remove it, we can always figure out what the relationship is between OpInfos and testing with/without cudnn is later.

Reply:
Interesting, I'll leave it for now if that's fine? I'm curious to see if any other OpInfo also needs to disable cuDNN for some tests, might be a good weekend investigation.

Reply:
Yup, let's leave this for now